packages/suite/src/components/suite/skeletons.ts

```diff
@@ -53,21 +53,32 @@ const shimmerEffect = css`
     background-size: 200%;
 `;
 
-interface SkeletonProps {
-    width?: string;
-    height?: string;
-    size?: string;
+type SkeletonBaseProps = {
     background?: string;
     animate?: boolean;
-    borderRadius?: string;
-    alignItems?: string;
-}
+};
+
+type SkeletonRectangleProps = SkeletonBaseProps & {
+    width?: string | number;
+    height?: string | number;
+    borderRadius?: string | number;
+};
+
+type SkeletonCircleProps = SkeletonBaseProps & {
+    size?: string | number;
+};
+
+const getValue = (value: string | number | undefined) => {
```

Comment:
I think it's OK to use both, because with this approach we can simulate how these styling props behaves in react in general. In my opinion props like these should work the same across whole app. 
 
You should be able to use just a number and in this case it means pixels, but sometimes you can use other units (%, em, etc.).

How it works in style prop in general:

```jsx
<SomeComponent 
  style={{
    width: 50
  }}
/>

<SomeComponent 
  style={{
    width: "50%"
  }}
/>
```

Reply:
Ahhh, got it, haven't noticed you are using values other than numbers!